### PR TITLE
Match design with colors and initial theming

### PR DESCRIPTION
**Pull request by hojberg**
## Overview
Match the UI to the design as much as possible given the missing elements.

Screen Shot 2021-02-08 at 16 49 49

## Implementation notes
* Add all colors from design
* Update fonts
* Add theming variables (supporting just the default theme for now)
* Update Sidebar and Workspace to reflect design and theme
* Remove unused UI elements and state transitions

## Loose ends
This is dependent on https://github.com/unisonweb/codebase-ui/pull/2

I don't love the icons from FontAwesome so I'll likely circle back with more custom ones.
Related; we are currently dependent on a web connection to load fonts and icons—I'd like to fully support offline mode in the future.

### CODE REVIEW

File: src/App.elm [hunk 1]
```diff
@@ -48,34 +47,14 @@ init _ =
...
-            in
-            ( { model | openDefinitions = nextOpenDefinitions }, Cmd.none )
-
-        ToggleShowCode hash ->
-            let
-                nextOpenDefinitions =
-                    model.openDefinitions
-            in
-            ( { model | openDefinitions = nextOpenDefinitions }, Cmd.none )
-
```
Comment (hojberg) [inline src/App.elm:78]: This weren't being used right now and was a bit presumptuous of the future implementation.

File: src/UI/Icon.elm [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    i [ class ("icon fas fa-" ++ name) ] []
+
+
+caretRight : Html msg
+caretRight =
+    icon "caret-right"
+
+
+caretDown : Html msg
+caretDown =
```
Comment (hojberg) [inline src/UI/Icon.elm:18]: I can imagine these being inlined svgs instead of FontAwesome in the near future

**Comment by hojberg** [comment]
The caretDown off center really bothers me...

**Comment by hojberg** [comment]
Screen Shot 2021-02-08 at 17 07 09

Improved it slightly.

**Review by pchiusano — APPROVED** [review]
Sweet!

**Pull request merged**